src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function convertToNumber(salary) {
+  const cleanedString = salary.replace(/[$,]/g, '');
+  const number = Number(cleanedString);
+
+  return number;
+}
+
+const ul = document.querySelector('ul');
+const liArr = [...ul.querySelectorAll('li')];
+
+function sortList(propName) {
+  liArr.sort((a, b) => {
+    const salaryB = convertToNumber(b.dataset[propName]);
+    const salaryA = convertToNumber(a.dataset[propName]);
+
+    return salaryB - salaryA;
+  });
+
+  ul.innerHTML = '';
+  liArr.forEach(li => ul.append(li));
+}
+
+sortList('salary');
+
+function getEmployees() {
```

Comment:
Shorten version:

```
function getEmployees() {
  const result = liArr.map(item => ({
      name: item.textContent.trim(),
      position: item.dataset.position,
      salary: item.dataset.salary,
      age: item.dataset.age,
    }));

  return result;
}
```

Reply:
Ok nice suggestions. Thanks. I've fixed it once again.